UsageStatistics_Plugin_Admin.php

```diff
@@ -105,15 +105,15 @@ public function w3tc_ajax_ustats_get() {
 	 * Ajax: Test access log path.
 	 */
 	public function w3tc_ajax_ustats_access_log_test() {
-		$nonce = isset( $_REQUEST['_wpnonce'][0] ) ? sanitize_key( $_REQUEST['_wpnonce'][0] ) : false;
+		$nonce = isset( Util_Request::get_array( '_wpnonce')[0] ) ? Util_Request::get_array( '_wpnonce' )[0] ) : false;
```

Comment:
Use `! empty()`.  You may want to set a variable to avoid calling twice.

Reply:
Changed to ! empty and created a variable to avoid 2 function calls.